application/language/english/ip_lang.php

```diff
@@ -1,6 +1,8 @@
 <?php
 
 $lang['cldr'] = 'en';
+//Day of the week start. 0 (Sunday) to 6 (Saturday)
+$lang['week_start'] = 0;
```

Comment:
Yes, you have true. I put it here because:
1. Translators 100% knows, how to set right default for specific language/nation.
2. Me as user, I expect, that after changing language, every nation/language specific features are set automatically. (It could be at settings too with override capability).

So I think at future releases currency, position of currency, decimal point... could be placed here too. Of course with ability to edit it by users. So, if no setting is at config (at database) default form language file is used. As you have it now at any OS.